### ISSUE TITLE
Image request randomly fails nightwatch test

### ISSUE THREAD

**Issue by windse7en** [cerner/terra-core#339]
### Description of Issue

Image http request in [here](https://github.com/cerner/terra-core/blob/master/packages/terra-image/tests/nightwatch/ImageDefault.jsx#L5) randomly cause nightwatch test failure. It doesn't get expected image in 3000 ms.

<img width="1077" alt="screen shot 2017-05-11 at 10 40 15 am" src="https://cloud.githubusercontent.com/assets/3724179/25958479/4d2caf54-3636-11e7-8b28-c72767bdbc27.png">

- [ ] Plan to store images in our source repo to fix this.

**Comment by Matt-Butler** [comment]
For this issue, we will also need to add a loader for images: https://webpack.js.org/guides/asset-management/#loading-images

**Comment by Matt-Butler** [comment]
For this issue to be resolved, we should be able to turn wifi off on our computer and the tests should all still pass.

**Comment by Matt-Butler** [comment]
When running nightwatch tests offline, I receive this errror for every package:
```
 FAIL  packages/terra-list/tests/jest/List.test.jsx
  ● Test suite failed to run

    proxies not supported on this platform. On v8/node/iojs, make sure to pass the --harmony_proxies flag
      
      at new global.Proxy (node_modules/harmony-reflect/reflect.js:2158:13)
      at Object.<anonymous> (node_modules/identity-obj-proxy/src/index.js:17:9)
      at Object.<anonymous> (packages/terra-list/node_modules/terra-base/lib/baseStyles.js:3:1)
      at Object.<anonymous> (packages/terra-list/src/List.jsx:4:1)
      at Object.<anonymous> (packages/terra-list/tests/jest/List.test.jsx:2:13)
      at handle (node_modules/worker-farm/lib/child/index.js:41:8)
      at process.<anonymous> (node_modules/worker-farm/lib/child/index.js:47:3)
      at emitTwo (events.js:87:13)
      at process.emit (events.js:172:7)
      at handleMessage (internal/child_process.js:686:10)
      at Pipe.channel.onread (internal/child_process.js:440:11)
```

**Comment by Matt-Butler** [comment]
PR: https://github.com/cerner/terra-core/pull/359

**Comment by Matt-Butler** [comment]
This issue should be resolved in #359.
